README.md

```diff
@@ -20,7 +20,7 @@ The `./scripts/run_tests.sh` script runs all unit tests using [py.test](http://p
 
 With the virtual environment active, run `python setup.py --version` to see the current version **on the current branch**.  Use `git tag` to add release tags to commits, and push the tags.
 
-You can reference these release tags in requirements files in other repositories.  See [notification-api](https://github.com/department-of-veterans-affairs/notification-api/blob/master/requirements-app.txt) for an example.
+You can reference these release tags in requirements files in other repositories.  See [notification-api](https://github.com/department-of-veterans-affairs/notification-api/blob/master/requirements-app.txt) for an example.  This isn't strictly necessary anymore because notification-api uses Poetry to manage dependencies.
```

Comment:
please remove the first sentence - that linked file no longer exists.
please add 'by running the command `poetry update notification-utils` when the utils PR is merged' to the second sentence.

```suggestion
After merging changes, to update the notification-api to use the updates, run `poetry update notification-utils` in an api branch and then push the PR for approval/merge. The PR should just containt chanegs made to the lock file.
```

Reply:
Changed.

Reply:
well, fix my typos if you take that suggestion.